# coding: utf-8
try:
    a = 1 / 0
#     arr = [2, 3]
#     print arr[3]
except Exception as e:
    print(e)
finally:
    print('end')
print('ddd')

